app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/CategoryTree.php

```diff
@@ -94,8 +94,17 @@ public function getTree(ResolveInfo $resolveInfo, int $rootCategoryId) : array
         $this->joinAttributesRecursively($collection, $categoryQuery);
         $depth = $this->depthCalculator->calculate($categoryQuery);
         $level = $this->levelCalculator->calculate($rootCategoryId);
+
+        // If root category is being filter, we've to remove first slash
+        if ($rootCategoryId == 1) {
```

Comment:
Looks like we need to use `\Magento\Catalog\Model\Category::TREE_ROOT_ID` instead of `1`

Reply:
that's exactly what I didn't found when I did that fix..
awesome! I will be pushing the update